composer.json

```diff
@@ -12,10 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^2.3"
+        "squizlabs/php_codesniffer": "^2.3",
+        "escapestudios/symfony2-coding-standard": "~2.0"
+    },
+    "suggest": {
+        "fabpot/php-cs-fixer": "^1.11 - For easy CS fixing"
```

Comment:
@shrikeh 

You could have a look at https://github.com/refinery29/php-cs-fixer-config to see how we're making use of `fabpot/php-cs-fixer`, maybe worth adding a `Makefile`, too?